code/modules/security_levels/keycard authentication.dm

```diff
@@ -186,13 +186,13 @@ var/global/maint_all_access = 0
 		for(var/obj/machinery/door/airlock/D in A)
 			D.emergency = 1
 			D.update_icon(0)
-	minor_announcement.Announce("The maintenance access requirement has been revoked on all airlocks.")
+	minor_announcement.Announce("Access Restrictions on Maintenance and External Airlocks have been removed.")
```

Comment:
The R shouldn't be capitalized. Probably no need to capitalize the airlock names either.